testdata/config/attributes.yaml

```diff
@@ -51,6 +51,8 @@ spec:
         valueType: STRING
       target.uid:
         valueType: STRING
+      destination.uid:
```

Comment:
Nit: this list was in alphabetical order before, can we maintain that? The connection attributes should be moved too.

Reply:
is it ok, not reorder it right now ?
We will do a reorder PR later :-)

Reply:
Yeah, not blocking.